scripts/variables.sh

```diff
@@ -23,4 +23,13 @@ TREE_WIDTH_OPTION="@sidebar-tree-width"
 
 SUPPORTED_TMUX_VERSION="1.9"
 
-SIDEBAR_DIR="$HOME/.tmux/sidebar"
+SIDEBAR_DIR_XDG="${XDG_CONFIG_HOME:-$HOME/.config}/tmux/sidebar"
+SIDEBAR_DIR_OLD="$HOME/.tmux/sidebar"
+if [ -d "$SIDEBAR_DIR_XDG" ]; then
+	SIDEBAR_DIR="$SIDEBAR_DIR_XDG"
+elif [ -d "$SIDEBAR_DIR_OLD" ]; then
+	SIDEBAR_DIR="$SIDEBAR_DIR_OLD"
+else
+	SIDEBAR_DIR="$SIDEBAR_DIR_XDG"
+fi
+unset SIDEBAR_DIR_XDG SIDEBAR_DIR_DEFAULT
```

Comment:
What do you think about creating a small function `sidebar_dir` for this?
The variables inside can be local `local sidebar_dir_xdg=...`. Then we could have only one global variable:

`SIDEBAR_DIR="$(sidebar_dir)"`

Reply:
✅ makes sense!